generators/client/templates/xamarin/src/Project.Client.Xamarin.Core/App.cs.ejs

```diff
@@ -0,0 +1,66 @@
+﻿<%#
+ Copyright 2013-2020 the original author or authors from the JHipster project.
+ This file is part of the JHipster project, see https://www.jhipster.tech/
+ for more information.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-%>
+
+using System;
+using Akavache;
+using <%= namespace %>.Client.Xamarin.Core.Models;
+using <%= namespace %>.Client.Xamarin.Core.Services;
+using <%= namespace %>.Client.Xamarin.Core.ViewModels;
+using MvvmCross;
+using MvvmCross.ViewModels;
+using System.Net.Http;
+using System.Reactive.Linq;
+using System.Threading.Tasks;
+using MvvmCross.Logging;
+
+namespace <%= namespace %>.Client.Xamarin.Core
+{
+    public class App : MvxApplication
+    {
+        public override void Initialize()
+        {
+            Akavache.Registrations.Start("<%= namespace %>");
+            var log = Mvx.IoCProvider.Resolve<IMvxLogProvider>().GetLogFor("<%= namespace %>");
+
+            var httpHandler = new HttpClientHandler();
+            httpHandler.ServerCertificateCustomValidationCallback = (o, cert, chain, errors) => true;
```

Comment:
we need to find a solution for this

Reply:
https://docs.microsoft.com/en-us/xamarin/cross-platform/deploy-test/connect-to-local-web-services